jbmc/src/miniz/miniz.h

```diff
@@ -173,6 +173,10 @@
 #pragma once
 
 
+#ifdef _MSC_VER
```

Comment:
This leaks into the scope of anyone including miniz.h -- if your intent is scope of this header file, use pragma-push / pragma-pop to scope it. (the .cpp file one is fine, implicitly scoped to the translation unit)